docs/install/manage-add-ons-packages.md

```diff
@@ -0,0 +1,425 @@
+extras = test
+```
+
+Stop the backend with {kbd}`ctrl-c`.
+Then apply your changes and start the backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+Depending on the package version you override, an upgrade step might need to be performed in the Plone control panel.
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+Follow the upgrade information, if present.
+
+
+(manage-build-and-start-your-instance-label)=
+
+### Build and start your instance
+
+Build and run the Plone backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+In a web browser, visit http://localhost:8080/ to see that Plone is running.
+
+Your instance is running in the foreground.
+
+
+(manage-installation-details-label)=
+
+## Backend installation details
+
+The `Makefile` at the root of you project invokes commands in `backend/Makefile`.
+Here are excerpts from `backend/Makefile` to show details of the `make build-backend` command.
+
+```makefile
+bin/pip:
+	@echo "$(GREEN)==> Setup Virtual Env$(RESET)"
+	python3 -m venv .
+	bin/pip install -U "pip" "wheel" "cookiecutter" "mxdev"
+
+instance/etc/zope.ini:	bin/pip
+	@echo "$(GREEN)==> Install Plone and create instance$(RESET)"
+	bin/cookiecutter -f --no-input --config-file instance.yaml https://github.com/plone/cookiecutter-zope-instance
+	mkdir -p var/{filestorage,blobstorage,cache,log}
+
+# ...
+
+.PHONY: build-dev
+build-dev: instance/etc/zope.ini ## pip install Plone packages
+	@echo "$(GREEN)==> Setup Build$(RESET)"
+	bin/mxdev -c mx.ini
+	bin/pip install -r requirements-mxdev.txt
+```
+
+The command `make build-backend`:
+
+-   Invokes the target `build-dev` target in `backend/Makefile`.
+-   This invokes the target `instance/etc/zope.ini`.
+-   This invokes the target `bin/pip`.
+
+    -   This creates a `Python` virtual environment if one does not exist.
+    -   It installs and upgrades Python package management tools in that virtual environment.
+
+-   Returning to the target `instance/etc/zope.ini`:
+
+    -   This creates or updates the Zope configuration from its `instance.yaml` file using `cookiecutter-zope-instance`.
+    -   Creates specified directories, if they do not exist.
+
+-   Returning to the target `build-dev`:
+
+    -   This generates the `mxdev` files as described above in {ref}`manage-mxdev-usage-overview-label`.
+    -   Installs Plone core packages and add-ons from the files generated by `mxdev`.
+
+You can configure your Zope instance as described in the section {ref}`manage-common-management-tasks-label`.
+
+
+````{todo}
+configuration frontend:
+
+- add an add-on
+- checkout an add-on for development with mrs.developer
+
+Add info here or link to respective frontend chapter.
+localhost:8050/volto/addons/index.html#configuring-a-volto-project-to-use-an-addon
+This already exists above:
+For developing add-ons for the Plone frontend, Volto, see {doc}`volto/addons/index`.
```

Comment:
@ksuess see the above two lines.

Is this todo needed here? I would prefer to remove it from the documentation, and create an issue.

Reply:
Yes, please.